fuel_surcharge.gemspec

```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |spec|
   # Specify which files should be added to the gem when it is released.
   # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
   spec.files = Dir.chdir(File.expand_path(__dir__)) do
-    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features|.github)/}) }
```

Comment:
Maybe we should have a `select()` instead of a `reject()` here to be safe and future proof ?

Reply:
Including all except Build/workflow files 
Vs
Include a big list of files and folders

Don’t know... really